changelog.txt

```diff
@@ -7659,7 +7659,7 @@ Add knobs to the [ColorIndicator Story](https://github.com/WordPress/gutenberg/p
 * Fix keyboard interaction (up/down arrow keys) causing focus to transfer out of the default block’s insertion menu.
 * Fix regression causing dynamic blocks not rendering in the frontend.
 * Fix vertical alignment issue on Media & Text block.
-* Fix some linter errors in master branch.
+* Fix some linter errors in main branch.
```

Comment:
Not sure if this is an oversight, or it's on purpose (because at the time, it wasn't called `trunk`?) :thinking: (If the latter, then we'd also need to change the other occurrences in the changelog, no?)

Reply:
I intentionally changed it to "main branch" to avoid ever having to change it again in the future, but that seems unlikely so I'll change it to be trunk explicitly.